src/plugins/workspace/public/components/workspace_fatal_error/workspace_fatal_error.tsx

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import {
+  EuiButton,
+  EuiEmptyPrompt,
+  EuiPage,
+  EuiPageBody,
+  EuiPageContent,
+  EuiCallOut,
+} from '@elastic/eui';
+import React from 'react';
+import { FormattedMessage } from '@osd/i18n/react';
+import { IBasePath } from 'opensearch-dashboards/public';
+import { useOpenSearchDashboards } from '../../../../opensearch_dashboards_react/public';
+import { formatUrlWithWorkspaceId } from '../../../../../core/public/utils';
+
+export function WorkspaceFatalError(props: { error?: string }) {
+  const {
+    services: { application, http },
+  } = useOpenSearchDashboards();
+  const goBackToHome = () => {
```

Comment:
Should it simply go back to `/` which might  be cleaner?

Reply:
In order to go back to homepage of default workspace, we need to:
1. Remove the workspace id inside the url path. (which will be done by second parameter of `formatUrlWithWorkspaceId`)
2. construct the path with potential basePath defined in yml file.

So I think the call to formatUrlWithWorkspaceId is necessary here.

Reply:
If we let it navigate to `/<basePath>`, then it should be able to automatically redirect to landing page, which by default is the `home` app, what do you think?

Reply:
Make sense, we should take `default route` into consideration.Thanks for catching this and updated accordingly.